packages/jest-runner/src/__tests__/testRunner.test.js

```diff
@@ -65,43 +65,6 @@ test('injects the serializable module map into each worker in watch mode', () =>
     });
 });
 
-test('does not inject the serializable module map in serial mode', () => {
```

Comment:
pretty sure this code path no longer exists with my changes, please sanity check my assumption